### ISSUE TITLE
parseArgs: fix 32/64 bit compatiblity in formatstrings for cross compiling

### ISSUE THREAD

**Issue by rschwebel** [Fiware/context.Orion-LD#1255]
Trying to print `int64_t` with `%ld` and `uint64_t` with `%lu` results in a compiler error:

`error: format '%lu' expects argument of type 'long unsigned int', but argument N has type 'uint64_t' {aka 'long long unsigned int'}`

when cross built for ARM 32 bit.

The correct way to print these types is with

```
  #include <cinttypes>
  ...
  printf("%" PRId64 "\n", var);

```
instead of `%ld` and

```
  printf("%" PRIu64 "\n", var);
```

instead of `%lu`.

MR will follow in a minute.

**Comment by kzangeli** [comment]
I believe this issue was addressed in the PR #1256

**Comment by rschwebel** [comment]
Thanks for your help!
